integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/merge/MergeTestCase.scala

```diff
@@ -842,7 +842,7 @@ class MergeTestCase extends QueryTest with BeforeAndAfterAll {
       Row("d", "3")
     ).asJava, StructType(Seq(StructField("key", StringType), StructField("value", StringType))))
 
-    initframe.write
+    initframe.repartition(1).write
```

Comment:
why this change

Reply:
it is needed or else spark will not be able to find the path to the carbon file